lib/api/controllers/adminController.js

```diff
@@ -119,11 +120,34 @@ function AdminController(kuzzle) {
         modifiedRequestObject = newRequestObject;
 
         _.forEach(kuzzle.pluginsManager.plugins, (object, id) => {
-          pluginsConfig[id] = _.assignIn({}, object);
-          if(pluginsConfig[id].object.context && pluginsConfig[id].object.context.config) {
+          pluginsConfig[id] = JSON.parse(stringifySafe(object));
```

Comment:
It was my first idea but doing this way (via %kuzzle/config) you loose the eventually computed / generated configuration (for example, kuzzle-plugin-cluster have a lot of things generated in it) then you loose the context too. 
Since this route aims to give some informations for troubleshoot purposes, displaying only what is inside %kuzzle/config is not enough in my opinion.